api/v1alpha1/helpers.go

```diff
@@ -11,3 +16,24 @@ func (a *AddonMetadataSpec) FromYAML(data []byte) error {
 func (a *AddonImageSetSpec) FromYAML(data []byte) error {
 	return yaml.Unmarshal(data, a)
 }
+
+// GetSemver - Returns the semver version matching "MAJOR.MINOR.PATCH".
+func (a *AddonImageSetSpec) GetSemver() (string, error) {
+	if parts := strings.SplitN(a.Name, ".", 2); len(parts) == 2 {
```

Comment:
Isn't this function too permissive?
An input like "x.a.v.c.x.x." would pass too where instead it should be raising error right?

wdyt of this suggestion (validates the to-be-returned semver string as well)
```
import "golang.org/x/mod/semver"
```
```suggestion
	if parts := strings.SplitN(a.Name, ".", 2); len(parts) == 2 && semver.IsValid(parts[1]) {
```

Reply:
done